generating_an_expression_matrix/README.md

```diff
@@ -61,98 +61,120 @@ This is what the your history will look like:
 
-In your example data you will see 'Homo_sapiens.GRCh38.95.gtf.gz', which is the human reference annotations as retrieved from Ensembl in GTF format, and a transcript-level description of the ERCC spike-ins (also in GTF format). We will use these to generate the transcript/ gene mapping, first by combining the two types of annotation and second by passing that information to a tool that extracts just the identifiers we need:
+In your example data you will see 'Homo_sapiens.GRCh38.95.gtf.gz', which is the human reference annotation as retrieved from Ensembl in GTF format. This annotation contains gene, exon, transcript and all sorts of other information on the sequences. In the example data, you will also see the annotations for the ERCC spike-ins (also in GTF format). We will use these to generate the transcript/ gene mapping, first by combining the two types of general annotation and second by passing that information to a tool that extracts just the transcript identifiers we need:
 
 Find the 'concatenate' tool in Galaxy under 'Text Manipulation':
 
 ![concatenating annotation files](concatenate.png)
 
-Select biological and technical sequences from your history, and use them as inputs. Run the tool and generate your combined annotations.
- 
-> EXERCISE: Which of the 'attributes' in the last column of the GTF files contains the transcript and gene identifiers?
+Select biological and spike-in sequences from your history, and use them as inputs. Run the tool and generate your combined annotations.  
+> EXERCISE: Which of the 'attributes' in the last column of the GTF files contains the transcript and gene identifiers? Hint: The file is organised such that the last column (headed 'Group') contains a wealth of information in the format: attribute1 "information associated with attribute";attribute2 "information associated with attribute 2" etc.
 
-Now we have combined annotations, we can parse the GTF file using the [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html) package in R. We've made a convenience tool in Galaxy to wrap that functionality for you- search for 'GTF2GeneList':
+Now we have combined annotations, we can parse the GTF file using the [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html) package in R. This parsing will give us an index with a list of transcript names and their corresponding gene names for counting. We've made a convenience tool in Galaxy to wrap that functionality for you- search for 'GTF2GeneList':
```

Comment:
Maybe 'index' is a bit overloaded here? 'Conversion table' instead?

Reply:
Yes!